css-two-column-layout/index.html

```diff
@@ -0,0 +1,95 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>CSS Challenge - Two Column Layout</title>
+  <link rel="stylesheet" href="fonts.css">
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+  <div class="container">
+    <div class="row">
+      <div class="column-full">
+        <h1>SCALE, SPEED, AGILITY</h1>
+        <div class="subheader">
+          <p>We've dedicated thousands of man hours to making healthcare
+            institutions in the Helium network run 10x more efficiently and
+            deliver an unparalleled quality of care.
+          </p>
+        </div>
+      </div>
+    </div>
+
+    <div class="row padding-row">
+      <div class="column-half">
+        <img class="icon" src="images/software.svg" alt="software">
```

Comment:
This is labeled as `column-half`, but this icon is definitely not taking up half of the overall space in the row 🤔 

This row would have four `column-half` elements, and each one should hold the image and the text for that chunk of the row. This image is an example of what one `column-half` in this layout should represent. You will need a `row` inside the `column-half`, though.
![image](https://user-images.githubusercontent.com/32880187/176976097-72ea44fc-1569-491b-9ddf-f6f81679c57f.png)